City-of-Pori/pori-kada
# PORI-135 Slideshow to match design

To test:

1. drush fra -y && drush cc all
2. Go to any attraction card and make sure the slideshow layout matches the designs.
3. Go to attraction card listing page and make sure each item has visible images and listing looks good.

Desktop design: https://drive.google.com/open?id=1_yY6yogj-ymzhbr49ZDnqg3vxyC0uGg2&authuser=0

Mobile design: https://drive.google.com/file/d/18U0JqCmH5TsgKZjy7nAUEydWceqBa5xb/view
